python/hsfs/feature_group.py

```diff
@@ -1343,6 +1354,11 @@ def _create_feature_monitoring(
                 Options are "HOURLY", "DAILY", "WEEKLY", "MONTHLY", defaults to "DAILY".
```

Comment:
In the JobScheduler constructor, it seems MONTHLY it is not supported, only HOURLY, DAILY and WEEKLY.